src/plugin/debugger/mhpdbgc.c

```diff
@@ -1312,13 +1312,39 @@ static void print_dscb(struct DSCB *dscb)
       dscb->size, dscb->stype, stnam);
 }
 
+static const char *hma_id_to_name(uint16_t id)
+{
+  static char buf[32];
+
+  switch (id) {
+    case 0x0000:
+      return "FREE";
+    case 0x0001:
+      return "DOS";
+    case 0xFF33:
+      return "IO.SYS";
+    case 0xFFFF:
+      return "MSDOS.SYS";
+  }
+  snprintf(buf, sizeof buf, "%04x", id);
```

Comment:
Would it be possible to resolve
the owner's name via its mcb,
rather than to just print on owner's
mcb address?

Reply:
Yes, sounds good.